build/bazel/remote/execution/v0_2/BUILD

```diff
@@ -0,0 +1,17 @@
+package(default_visibility = ["//visibility:public"])
+
+# load("@protobuf//:protobuf.bzl", "proto_library")
```

Comment:
Wait, why isn't this needed?

Reply:
The `proto_library` rule is builtin to Bazel; I originally thought I had to load it (it does depend on the protoc repository though, which must be explicitly specified in `WORKSPACE`). I'll remove this line.